c7n/resources/emr.py

```diff
@@ -75,6 +75,23 @@ def augment(self, resources):
 
 @actions.register('terminate')
 class Terminate(BaseAction):
+    """Action to terminate EMR cluster(s)
+
+    It is recommended to apply a filter to the terminate action to avoid
+    termination of all EMR clusters
+
+    :example:
+
+        .. code-block: yaml
+
+            policies:
+              - name: emr-terminate
+                resource: emr
```

Comment:
i think we need to document emr with query filters on state, as the service keeps history of previous job flow clusters.